src/main/deploy/swerve/modules/physicalproperties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "optimalVoltage": 12,
+  "robotMass": 115,
+  "wheelGripCoefficientOfFriction": 1.85,
```

Comment:
May need to update this? I'm not sure where we might be able to find the CoF of the tread we use